### PR TITLE
[584] Fix bug on the edge label font size after refresh

### PR DESCRIPTION
**Pull request by scosta-obeo**
This commit also refactor class FontFormatUpdater and update the tests to check also the edge labels.

Bug: https://github.com/eclipse-sirius/sirius-desktop/issues/584

### CODE REVIEW

File: plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/internal/refresh/listeners/FontFormatUpdater.java [hunk 1]
```diff
@@ -91,34 +130,119 @@ public boolean isAggregatePrecommitListener() {
...
-            super(domain);
-            this.style = style;
+    /**
+     * Return the list of changed feature on FontStyle during the transaction grouped by FontStyle.
+     * 
+     * @param event
+     *            the event of the transaction, containing the list of notification
+     * @return a map with the FontStyle object as key and a set of its modified features as value
+     */
+    private Map<FontStyle, Set<EStructuralFeature>> getFontStyleFeatueChanges(final ResourceSetChangeEvent event) {
```
Comment (lfasani) [inline plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/internal/refresh/listeners/FontFormatUpdater.java:140]: getFontStyleFeatureChanges
